draft-specifications/Netlayer-Unix-Domain-Socket.md

```diff
@@ -0,0 +1,123 @@
+# Unix domain socket netlayer documentation
+
+The Unix domain socket netlayer uses anonymous domain sockets to allow peers to
+communicate with one another. These sockets provide a fast and convenient
+mechanism for processes on the same machine to communicate with one another. The
+netlayer was designed from the point of view that the netlayer itself does not
+need any file system access, because of this netlayers need one or more
+introduction servers which facilitating new connections between peers, but do
+not relay the traffic once two peers have been introduced. Connections between
```

Comment:
The sentence above should be broken up into two or more. I think it would be worth mentioning the ocap reasons behind *why* we can't have the netlayer referring to file names, to educate readers who don't yet know the security issues with doing so.